parlai/core/build_data.py

```diff
@@ -380,6 +380,8 @@ def _unzip(path, fname, delete=True):
                 PathManager.mkdirs(outpath)
                 continue
             logging.debug(f"Extracting to {outpath}")
+            if '__MACOSX' in member:
```

Comment:
It logically makes sense to submit this as its own patch. Please remove it from this PR and add is as a separate one.